decorator/trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base'
+
+class TrimDecorator < Decorator
+  def correct_name
+    super.slice(0, 9) if super.length > 10
+  end
+end
```

Comment:
- [ ] This method returns the right result but it returns a result **_only_** if `super.length > 10`. That means if you passed a string that is less than 10 characters long, it won't return anything. Kindly fix this behavior.

Reply:
@ShadyShawkat Thank you for your constructive feedback.